#!/usr/bin/env python

# set up parameters that we care about
PACKAGE = 'jsk_pcl_ros'

from dynamic_reconfigure.parameter_generator_catkin import *;

from math import pi

gen = ParameterGenerator ()

## configured by heightmap creator
#gen.add("min_x", double_t, 0, "", -2.0, -10.0, 10.0)
#gen.add("max_x", double_t, 0, "", 2.0, -10.0, 10.0)
#gen.add("min_y", double_t, 0, "", -2.0, -10.0, 10.0)
#gen.add("max_y", double_t, 0, "", 2.0, -10.0, 10.0)
gen.add("use_offset", bool_t, 0, "", False)
gen.add("use_bilateral", bool_t, 0, "", False)
gen.add("bilateral_filter_size", int_t, 0, "", 5, 1, 50)
gen.add("bilateral_sigma_color", double_t, 0, "", 0.04, 0.0, 1.0)
gen.add("bilateral_sigma_space", double_t, 0, "", 5, 0.0, 100)

exit (gen.generate (PACKAGE, "jsk_pcl_ros", "HeightmapTimeAccumulation"))
